vispy/plot/plotwidget.py

```diff
@@ -162,7 +162,7 @@ def _configure_3d(self):
 
         self._configured = True
 
-    def histogram(self, data, bins=10, color='w', orientation='h'):
+    def histogram(self, data=None, bins=10, color='w', orientation='h'):
```

Comment:
Do you want to add the other new kwarg or a `**kwargs` (I'm usually no a fan but this might have to be an exception) too?

Also, could you update the docstring to mention None?